pkg/util/util.go

```diff
@@ -50,3 +57,32 @@ func EqualStringSlice(a, b []string) bool {
 	}
 	return true
 }
+
+func GetCmdLine(pid int) (string, error) {
+	data, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/cmdline", pid))
+	if err != nil {
+		return "", err
+	}
+	result := string(bytes.ReplaceAll(data, []byte("\x00"), []byte(" ")))
+	return result, err
+}
+
+func RemoveEmptyStrings(s []string) []string {
+	var r []string
+	for _, str := range s {
+		if str != "" {
+			r = append(r, str)
+		}
+	}
+	return r
+}
+
+func GetProcessPid(search string) (string, error) {
```

Comment:
would it be better to return a []int of pids instead of requiring the consumer to handle trimming, atoi, etc?

Reply:
also simplifies error handling for consumer